docs/faq.md

```diff
@@ -79,23 +79,31 @@ good to them.
 
 Probably yes. We run private experiments for this purpose.
 Please reach out to us at fuzzbench@google.com. If we agree to benchmark your
-fuzzer, please follow the guide on
-[adding a new fuzzer]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/)
-on how to integrate your fuzzer with FuzzBench.
+fuzzer, then proceed to these next steps:
 
-You can ignore the sections on [Requesting an experiment]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#requesting-an-experiment) and
+1. Follow the guide on
+[adding a new fuzzer]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/)
+on how to integrate your fuzzer with FuzzBench. You can ignore the sections on
+[Requesting an experiment]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#requesting-an-experiment) and
 [Submitting your integration]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#submitting-your-integration).
-Please test your fuzzer works with our benchmarks, we don't have CI to verify
-this for private experiments.
-Ideally, you should test all benchmarks using `make -j test-run-$FUZZER-all`.
-This takes too long on most machines so you should at least test a few of them:
-```
-make test-run-$FUZZER-zlib_zlib_uncompress_fuzzer test-run-$FUZZER-libpng-1.2.56
-```
 
-You should also run `make presubmit` to validate the fuzzer's name and
-integration code.
-When your fuzzer is ready, send us a patch file that applies cleanly to
+1. Run `make presubmit` to validate the fuzzer's name and integration code.
+
+1. Test your fuzzer works with our benchmarks as we don't have CI to verify
+   this for private experiments. Ideally, you should test all benchmarks using
+   `make -j test-run-$FUZZER-all`. This takes too long on most machines, so you
+   should at least test a few of them:
+   ```
+   make test-run-$FUZZER-zlib_zlib_uncompress_fuzzer test-run-$FUZZER-libpng-1.2.56
+   ```
+
+1. Run an hour-long local experiment with 2 fuzzers - your fuzzer and
```

Comment:
Not sure this really helps. The issues plaguing me have been build failures on some of the benchmarks. So doing an experiment with some benchmarks may not catch anything.

Reply:
Ok gotcha, closing then.